main/guides/getting-started/sell-concert-tickets-contract-explainer.md

```diff
@@ -1,11 +1,24 @@
 # Sell Concert Tickets Smart Contract
+
 This smart contract is designed to mint and sell event tickets as non-fungible tokens (NFTs) in the form of a semi-fungible asset. In this example there are three categories or classes of tickets:
 - Tickets near the front are the most expensive
 - Tickets in the middle rows are priced between expensive and cheap seats
 - Tickets in the back are the lowest priced
 
+> Note: This contract simulates traditional trading i.e. between a vendor and a consumer.
+
+## Objective
+
+The objective of this tutorial is to teach you the following:
+
+- A fundamental guide on how to **establish a smart contract**.
+- Explain the process of **initiating asset trading** within the Agoric environment including listing of assets and setting prices etc.
+- **Develop a trade handler** that manages the execution of business logic behind trades.
+
 ## Contract Setup
-The contract is parameterized with an `inventory` object, which contains information about the different types of tickets available for sale, their prices `tradePrice`, and the maximum number of tickets for each type `maxTickets`. For example:
+
```

Comment:
@ansabgillani can you remove extra lines here and everywhere else?

Reply:
fixed.